corehq/apps/indicators/utils.py

```diff
@@ -1,10 +1,10 @@
 from couchdbkit import ResourceNotFound
 from django.conf import settings
-from corehq.util.quickcache import quickcache
+from corehq.util.quickcache import skippable_quickcache
 from dimagi.utils.couch.database import get_db
 
 
-@quickcache([], timeout=60)
+@skippable_quickcache([], lambda: settings.UNIT_TESTING, timeout=60)
```

Comment:
traced the test failure to this function. some of the new data interfaces test were invoking it deep within the call stack and by doing so, the return value was being cached for 60 seconds.  since the mvp_docs tests seem to run right after the data interfaces ones (and they also rely on the return value of this function), it was returning the cached value and failing.